packages/visualizations/addon/components/number-format-selector.js

```diff
@@ -46,15 +46,15 @@ export default Component.extend({
      * @action updateFormat
      */
     updateFormat(format) {
-      this.attrs.onUpdateFormat({ format });
+      this.attrs.onUpdateFormat(format);
```

Comment:
Now you don't need these actions, you could 
`(action this.attrs.onUpdateFormat format)` in the template.

Same thing for clearFormat.